dunning_kruger.Rmd

```diff
@@ -103,3 +103,7 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# arbitrary change
```

Comment:
This change does look a little arbitrary.  Can you think of anything less arbitrary to add?

Reply:
Ok I'll have a think.